apps/studio/components/interfaces/Home/Connect/content/prisma/content.tsx

```diff
@@ -23,7 +23,7 @@ const ContentFile = ({ connectionStringPooler, connectionStringDirect }: Content
 DATABASE_URL="${connectionStringPooler}"
 
 # Direct connection to the database. Used for migrations.
-DIRECT_URL="${connectionStringDirect}"
+DIRECT_URL="${connectionStringPooler.replace('6543', '5432')}"
```

Comment:
Same here, it would be better if we'd pass in variables that contain the pooler mode and then we just pick the one we want here, instead of random port replacements (who knows what they mean)